src/features/upload/components/AdditionalUploadBtn.tsx

```diff
@@ -0,0 +1,23 @@
+import type { ComponentProps } from "react";
+
+import { Icon } from "@/common/components/Icon";
+
+const uploadBtnStyle = {
+  focus: "focus:border-gray-100 focus:text-gray-600 [&_*]:focus:stroke-gray-600",
+  hover: "hover:border-gray-100 hover:text-gray-600 [&_*]:hover:stroke-gray-600",
+  active: "active:border-gray-400 active:text-gray-800 [&_*]:active:stroke-gray-800",
+  disabled: "disabled:border-gray-400 disabled:text-gray-300 [&_*]:disabled:stroke-gray-300",
+};
+export const AdditionalUploadBtn = ({ className, ...rest }: ComponentProps<"button">) => {
+  return (
+    <button
+      {...rest}
+      className={`flex w-fit gap-6 rounded-26 border border-gray-300 bg-white px-24 py-14 text-16-semibold-140 text-gray-700 ${Object.values(
+        uploadBtnStyle,
+      ).join(" ")} ${className}`}
+    >
```

Comment:
@SeieunYoo @elbica 

이 PR에서는 안해도되는데 후에 [`clsx`](https://www.npmjs.com/package/clsx) 도입하는거 어때?
[`classnames`](https://www.npmjs.com/package/classnames)랑 api는 비슷한데 [벤치마크가 `clsx`가 더 좋기도하구](https://github.com/lukeed/clsx/tree/master/bench).. 저도 써봤는데 러닝커브는 없을 듯! 조건부나 복잡한 classname 작성할 때 좀 더 편하고 `.join(' ')` 같은거 안해도 돼

```suggestion
    <button
      {...rest}
      className={clsx("flex w-fit gap-6 rounded-26 border border-gray-300 bg-white px-24 py-14 text-16-semibold-140 text-gray-700", Object.values(uploadBtnStyle), className)}
    >
```

Reply:
오 테일윈드랑 잘 맞을 것 같네요👀

Reply:
이거는 @SeieunYoo 이 작업할 때 classname 복잡해지면 한번 적용해보자